.github/workflows/pr.yaml

```diff
@@ -64,12 +64,9 @@ jobs:
   wheel-build:
     needs: checks
     secrets: inherit
-    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@branch-23.08
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-build.yml@local-script
     with:
       build_type: pull-request
```

Comment:
`build_script` is now compulsory to be supplied to this workflow. Please add that